src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -785,4 +785,27 @@ protected function get_mobile_messages() {
 		 */
 		return apply_filters( 'tribe_events_views_v2_month_mobile_messages', $mobile_messages, $this );
 	}
+
+	/**
+	 * Do a short query (one event) to determine if we should add a noindex meta tag to the page.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Repository|false $events     The events repository. False by default.
+	 * @param DateTime                $start_date The start date (object) of the query.
+	 * @param Tribe__Context          $context    The current context.
+	 *
+	 * @return Tribe__Repository|false $events     The events repository results.
+	 */
+	public static function get_noindex_events( $events, $start_date ) {
+		$interval = new \DateInterval( 'P1M' );
+
```

Comment:
Potential here to add the view's repository args as well 🤔

Reply:
Yeah, shouldn't this run after the view has gotten it's events, and just reference those results? Is that possible (the view hook fires in time for the noindex meta tag hooks)? Would avoid the guesswork.

Maybe I'm missing something?

Reply:
It can't run after the view query - that doesn't happen until the content starts.
I am doing something to utilize the existing args though, have to test.